### DISCUSSION TITLE
Bad gateway error 502 out of the blue

### DISCUSSION THREAD

**Discussion by poldiderjungdrache** [nextcloud/all-in-one#3546] [Questions] [answered]
### Steps to reproduce
Not sure how to reproduce it since I made no changes

### Actual behavior 
I had set up Nextcloud AIO some 4 weeks back (nginxproxy, cloudflare proxy). It was running fine without any issues since about a week ago. Now I am suddenly getting bad gateway 502 when I try to exit it. I made no changes to the config prior.

### Host OS 
Ubuntu 22.04

#### Nextcloud AIO version 
7.2.1

#### Current channel 
latest
#### Other valuable info 
To troubleshoot I did upgrade to the latest version 7.2.1. However that did not fix it.
I found this [issue](https://github.com/nextcloud/all-in-one/discussions/3508) and also try 7.4.1 beta but that also did not help.

 Mastercontainer says all is green:
![Screenshot 2023-10-15 at 21 41 33](https://github.com/nextcloud/all-in-one/assets/59320185/9780cf85-0a3f-45e7-928f-8a94c4d73210)

I don't see anything obvious in the logs:
[nextcloud-aio-mastercontainer.txt](https://github.com/nextcloud/all-in-one/files/12910514/nextcloud-aio-mastercontainer.txt)
[nextcloud-aio-apache.txt](https://github.com/nextcloud/all-in-one/files/12910515/nextcloud-aio-apache.txt)
[nextcloud-aio-nextcloud.txt](https://github.com/nextcloud/all-in-one/files/12910516/nextcloud-aio-nextcloud.txt)

Any idea what the issue might be?

**Comment by szaimen** [comment]
Hi, can you post your nginx config here?

**Comment by poldiderjungdrache** [comment]
Hi,
thanks for the quick reply. 
I believe the nginxproxymanager is like described. 
Here you go this is what I set:
![Nginx Proxy Manager2](https://github.com/nextcloud/all-in-one/assets/59320185/05d6d377-c9f2-47fc-938d-61a3fa4b87ad)
![Nginx Proxy Manager1](https://github.com/nextcloud/all-in-one/assets/59320185/19d6caac-6abf-443f-982d-8f1f6e0bf767)
![image](https://github.com/nextcloud/all-in-one/assets/59320185/8f2fdccf-ef51-4316-b510-48a91f7a8dd7)

**Comment by szaimen** [comment]
I need the `Forward hostname / IP` and also the docker-compose of nginx proxy manager. Can you post both here?

**Comment by poldiderjungdrache** [comment]
The 'Froward hostname / IP' is http://nextcloud-aio.tianmimi.dmz:11000 (so the proxy is running on another host as nextcloud).
docker compose.yml:
[docker-compose.txt](https://github.com/nextcloud/all-in-one/files/12931348/docker-compose.txt)

Note: SKIP_DOMAIN_VALIDATION=true was added after the issue occurred (for testing) but it had no effect.

**Comment by szaimen** [comment]
> The 'Froward hostname / IP' is http://nextcloud-aio.tianmimi.dmz:11000

So it is running on a different host and going out to the public internet without encrypting the transport via tls or vppn? You know that this is completely insecure?

**Reply by poldiderjungdrache** [reply, marked as answer]
Don't worry it is strictly inside the DMZ. tianmimi.dmz is not a domain that makes any kind of sense on the public internet. 
But actually you have a point replacing this with the proper IP, i.e. 192.168.100.212, is a better idea. 
And you know what? After the change it is working again.
So if you have wired issues it is always DNS!? 
Darn it ... appologize for the disturbance. Thanks for the help regardless.

**Reply by szaimen** [reply]
all right! Good that you found the problem!
